docs/development/installation.md

```diff
@@ -10,43 +10,43 @@
 
 - Create a separate PostgreSQL user for the development and test environments. As per our policy, PostgreSQL user name must be the same as the application name: `nokul`.
 
-```bash
-sudo -u postgres psql <<-EOF
-        CREATE USER nokul WITH ENCRYPTED PASSWORD 'nokul';
-        ALTER ROLE nokul LOGIN CREATEDB SUPERUSER;
-EOF
-```
+  ```bash
```

Comment:
Markdown'la ilgili küçük bir bir bilgilendirmeyi komit mesajında yazdım.  Bu değişikliğin etkisini göstereyim (liste girintilerine dikkat):

#### Düzeltilmeden önce

![image](https://user-images.githubusercontent.com/47842/44983252-4058bf00-af81-11e8-913f-882b2ba44b1d.png)

#### Düzeltildikten sonra

![image](https://user-images.githubusercontent.com/47842/44983275-4e0e4480-af81-11e8-8b0f-6909ea3021d6.png)